### PR TITLE
feat(meta): fix bug of lease

### PR DESCRIPTION
**Pull request by Little-Wallace**
Signed-off-by: Little-Wallace <bupt2013211450@gmail.com>

I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

close https://github.com/singularity-data/risingwave/issues/3398
and add test for lease.

## Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

File: src/meta/src/rpc/server.rs [hunk 1]
```diff
@@ -466,3 +473,54 @@ pub async fn rpc_serve_with_store<S: MetaStore>(
...
     Ok((join_handle, shutdown_send))
 }
+
+#[cfg(test)]
+mod tests {
+    use std::thread::sleep;
+
+    use super::*;
+
+    #[tokio::test(flavor = "multi_thread")]
```
Comment (skyzh) [inline src/meta/src/rpc/server.rs:484]: Better not to have sleep in unit tests :) Ignoring this test looks like a better choice.

Comment (skyzh) [reply src/meta/src/rpc/server.rs:484]: Also tokio::sleep instead of thread::sleep?

**Comment by twocode** [comment]
cc @vdnhi for hitting this.

File: src/meta/src/rpc/server.rs (hunk 1 above)
Comment (Little-Wallace) [reply src/meta/src/rpc/server.rs:483]: good idea.

**Review by yezizp2012 — APPROVED** [review]

**Pull request merged**